plugins/polio/serializers.py

```diff
@@ -874,7 +874,7 @@ class NestedScopeSerializer(RoundScopeSerializer):
             class NestedGroupSerializer(GroupSerializer):
                 class Meta:
                     model = Group
-                    fields = ["id"]
+                    fields = ["id", "org_units"]
```

Comment:
Isn't this too costly?  Wouldn't this end up serializing all districts for the AFRO zone on each calendar refresh?

Reply:
This is a lot of integer to send through the api while there is no need for all the details 
<img width="321" alt="CleanShot 2023-07-05 at 22 06 10@2x" src="https://github.com/BLSQ/iaso/assets/185797/18736174-c63f-4941-9b47-f5b66932f858">

Reply:
Updated to use values from backend. the PR now depends on POLIO-846